data/lib/core/creature.lua

```diff
@@ -50,3 +50,48 @@ end
 function Creature.isTile(self)
 	return false
 end
+
+function Creature:setMonsterOutfit(monster, time)
+	if not self then
+		return false
+	end
```

Comment:
Is this even possible? Reaching here with null `self`? The same for the other functions.